MyHeartCounts/Resources/Assets.xcassets/image4.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
We should probably just use a single size asset here and for others. Reduces the repo size and these are only used for debugging now ...

Reply:
good catch i'll remove this (there's a couple other things as well that i wanna get rid of before merging this, eg the de.lukaskollmer-scoped bundle identifier). i was also planning on trying to have it pass as many GH tests as possible. if we fix all the linting issues (i might switch todos over to warnings in swiftlint temporarily, at least very early on) and have empty, always-passing tests this probably shouldn't be too difficult

Reply:
Sounds great; thank you!